tasks/Module 2/task_1.py

```diff
@@ -0,0 +1,59 @@
+"""Work with inventory."""
+
+
+def display_inventory(inventory):
+    """Print inventory."""
+    print('Inventory:')
+    items = 0
+    # check that inventory is a dict
+    if not isinstance(inventory, dict):
+        print('Wrong inventory type. Should be dict')
+        return
+    # print inventory and calculate sum of items
+    for key, value in inventory.items():
+        print(value, key)
+        items += value
+    print()
+    # check number of item and print according caution if needed
+    print('Total number of items:', items, end='\n\n')
+    if 60 <= items <= 69:
+        print('CAUTION: Your backpack weighs a lot, your stamina runs out quicker!')
+    elif 70 <= items <= 79:
+        print('CAUTION: Your equipment is very heavy, you\'re moving slower than usual!')
+    elif items >= 80:
+        print('CAUTION: You are overloaded, can\'t move!')
+
+
+def add_to_inventory(inventory, added_items):
+    """Add items from list to inventory."""
+    skipped = {}
+    added = 0
```

Comment:
I would rather rename variable `added` to indicate it is number of added items eg. `added_items_number` or `added_count` or something like that. Currently variable `skipped` is a dictionary, but similarly named `added` is integer, which may cause confusion.